src/workspaces/features/RunPipelineDialog/ParameterField.tsx

```diff
@@ -48,6 +48,7 @@ const ParameterField = (props: ParameterFieldProps) => {
       <Select
         onChange={handleChange}
         value={value}
+        required={parameter.required ?? true}
```

Comment:
Could you replace all the required by this:
```suggestion
        required={Boolean(parameter.required)}
```

It was meant to be a temporary hack but we can remove this now.

Reply:
Got it.